package.json

```diff
@@ -77,7 +77,9 @@
     "test": "jest",
     "dev": "rollup -c -w",
     "build": "rollup -c && ./scripts/update-readme.sh",
-    "lint": "npx prettier --write ."
+    "lint": "npx prettier --write .",
+    "fork": "npx ts-node ./src/tools/run-moonbeam-fork.ts --base-path /tmp/fork-data/ -n moonbeam",
+    "fork-solo": "npx ts-node ./src/tools/run-moonbeam-fork.ts --base-path /tmp/fork-data/ -n moonbeam --solo"
```

Comment:
Not sure we need another package script just for 1 argument.

Reply:
Done